server/gsheets.js

```diff
@@ -233,6 +238,44 @@ const retrieveAllReminders = async auth => {
   });
 };
 
+const retrieveAllBirthdays = async auth => {
+  const birthdays = [];
+  const SHEET_NAME = 'ET_birthdays';
+  const sheets = google.sheets({
+    version: 'v4',
+    auth
+  });
+
+  return new Promise(function(resolve, reject) {
+    sheets.spreadsheets.values.get(
+      {
+        spreadsheetId: SPREADSHEET_ID,
+        range: `${SHEET_NAME}`
+      },
+      (err, res) => {
+        if (err) return console.log(`The API returned an error: ${err}`);
+
+        const rows = res.data.values;
+
+        if (rows.length) {
+          rows.map(row => {
+            //filter to make sure that only people with consent to "Other communication"
+            if (row[4].toUpperCase() === 'YES') {
```

Comment:
Think using filters would convey this better? reduces few lines and is quite readable IMO
```js
rows.filter(row => row[4].toUpperCase() === 'YES')
        .map(row => birthdays.push({ name: row[0], date: row[1] }));
```

Reply:
Totally agree

Reply:
Thanks